api/controllers/ApiController.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * ApiController
+ *
+ * @description :: Server-side logic for managing Apiv1s
+ * @help        :: See http://sailsjs.org/#!/documentation/concepts/Controllers
+ */
+
+module.exports = {
+
+  /**
+   * `UserController.index()`
+   */
+  index: function (req, res) {
+    return res.json({
+      v1: sails.getBaseUrl() + sails.getUrlFor('v1/ApiController.index'),
```

Comment:
```
warn: sails.getBaseUrl() is deprecated and will be removed in Sails v1.0.  See http://sailsjs.org/documentation/reference/application/sails-getbaseurl for more info.
```

Link in warning does not work. Try: http://sailsjs.org/documentation/reference/application/sails-get-base-url

> This method is deprecated and will likely be removed or changed in Sails v1.0
>
> There is no reliable, cross-platform way to automatically detect the external URL of a running Sails app (or any other Node app). In mission-critical situations, you are advised to pre-determine the URL and save it in a custom environment-dependent configuration value (e.g. sails.config.appUrl) that you can reference elsewhere in the app.

For what it's worth, this issue is resolved in `mb-logging-api` with:
```
/**
 * Utility to provide reference to the root of the application. Used by files
 * in subdirectories to reference other files within the application directory
 * structure.
 *
 * Reference: https://gist.github.com/branneman/8048520#7-the-wrapper
 */
global.rootRequire = function(name) {
  return require(__dirname + '/' + name);
}
```
https://github.com/DoSomething/Quicksilver-Node/blob/master/mb-logging-api/mb-logging-api-server.js#L9-L18